src/game/server/entity.h

```diff
@@ -21,7 +21,6 @@ class CEntity
 {
 	MACRO_ALLOC_HEAP()
 
-private:
```

Comment:
This change does not seem to be needed anymore. Changes to files that exist in ddnet code base should be kept to a minimum to avoid git conflicts when updating.